terraform_compliance/main.py

```diff
@@ -56,6 +57,12 @@ def cli(arghandling=ArgHandling(), argparser=ArgumentParser(prog=__app_name__,
 
     steps_directory = os.path.join(os.path.split(os.path.abspath(__file__))[0], 'steps')
 
+    python_version_check()
+
+
+
+
+    
```

Comment:
There are a couple more empty lines than needed.

Reply:
Fixed in 61b8f56